File: 000/DeHSGTinhQN/1. Tong chu so x64/SUMN.py
def SUM(a):
    total = 0
    while a != 0:
        total += a % 10
        a //= 10
    return total

with open ("SUMN.INP", "r") as f:
    a = int(f.readline())

with open ("SUMN.OUT", "w") as f:
    f.write(str(SUM(a)))

File: 000/DeHSGTinhQN/1. Tong chu so x64/SUMN.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string N;
    freopen("SUMN.INP", "r", stdin);
    freopen("SUMN.OUT", "w", stdout);
    cin >> N;
    int sum = 0;
    for (int i = 0; i < N.size(); ++i) {
        sum += (N[i] - '0');
    }
    cout << sum;

    return 0;
}
